icu4c/source/i18n/units_converter.h

```diff
@@ -155,8 +155,7 @@ class U_I18N_API UnitsConverter : public UMemory {
      * @param targetIdentifier represents the target unit identifier.
      * @param status
      */
-    UnitsConverter(const char* sourceIdentifier, const char * targetIdentifier,
-     UErrorCode &status);
+    UnitsConverter(const StringPiece sourceIdentifier, StringPiece targetIdentifier, UErrorCode &status);
```

Comment:
optional: StringPiece is a value type, we don't usually bother labelling them const. And the const is inconsistent with the implementation in units_converter.cpp.

```suggestion
    UnitsConverter(StringPiece sourceIdentifier, StringPiece targetIdentifier, UErrorCode &status);
```

Reply:
done